DOCS-10084 landing page

wwarner-salesforce opened the pull request:
DOCS-10084 landing page

wwarner-salesforce committed:
DOCS-10084 landing page

DOCS-10084 landing page

wwarner-salesforce committed:
Update index.adoc

wwarner-salesforce committed:
Update index.adoc

wwarner-salesforce committed:
Update index.adoc

wwarner-salesforce committed:
Update index.adoc

wwarner-salesforce committed:
Merge branch 'latest' into docs-10084-landing-page

wwarner-salesforce committed:
Update nav.adoc

wwarner-salesforce committed:
Create asset-types.adoc

wwarner-salesforce committed:
Update nav.adoc

wwarner-salesforce committed:
Update nav.adoc

wwarner-salesforce committed:
Update asset-details.adoc

wwarner-salesforce committed:
Update index.adoc

wwarner-salesforce committed:
Add files via upload

wwarner-salesforce committed:
Update index.adoc

wwarner-salesforce committed:
Merge branch 'latest' into docs-10084-landing-page

wwarner-salesforce committed:
Merge branch 'latest' into docs-10084-landing-page

wwarner-salesforce committed:
Update asset-types.adoc

JennyHajee commented on modules/ROOT/pages/index.adoc:
```diff
@@ -5,71 +5,19 @@ endif::[]
 
 Using multiple languages with Unicode is supported fully throughout Exchange and in all metadata fields, including asset names, descriptions, markdown pages, categories, tags, and reviews.
 
-[[content-types]]
```
hi, is the removed content no longer applicable or did you just streamline the landing page since the content is located in another topics?

wwarner-salesforce replied:
This PR moves the "Content Types" list from the landing page to the new page "Asset Types". It never really made sense for it to be part of the landing page.

rithishapadmanabh replied:
Callout - If the asset types section was linked anywhere else in the product/documentation it will need to be updated to point to the new section instead.

JennyHajee reviewed (commented):
Is the removed content no longer valid? Or is this info available in other topics?

rithishapadmanabh commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,6 +4,28 @@ include::_attributes.adoc[]
+
+To view assets:
+
+. From the public https://www.anypoint.mulesoft.com/exchange/[Exchange], click an asset and view its description.
```
Re> "public" Exchange? It should just be from 'Exchange'. We are trying to move away from the Public vs Private Exchange nomenclature especially in external facing content. Would word it as 'From Exchange'

rithishapadmanabh replied:
Also should we consider linking to the Explore Asset Information section from here given that sections provides details on how to read the asset details page in detail?

rithishapadmanabh commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,6 +4,28 @@ include::_attributes.adoc[]
+To view assets:
+
+. From the public https://www.anypoint.mulesoft.com/exchange/[Exchange], click an asset and view its description.
+. Click *Helpful Links* in the navigation area for links to documentation.
```
This is not accurate.. not every asset has 'helpful links' the navigation area. E.g.: https://www.anypoint.mulesoft.com/exchange/com.mulesoft.connectors/mule-sap-s4hana-cloud-connector/

rithishapadmanabh replied:
Can we say something on the lines of 'View the documentation and metadata to learn more about the asset'

JennyHajee replied:
Yes, I can change this.

rithishapadmanabh commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,6 +4,28 @@ include::_attributes.adoc[]
+
+. From the public https://www.anypoint.mulesoft.com/exchange/[Exchange], click an asset and view its description.
+. Click *Helpful Links* in the navigation area for links to documentation.
+. Click *Login* to view a private Exchange for your organization.
```
Once again, would refrain from using private exchange. Suggest something on the lines of 'Click Login to view assets published into Exchange by your organization.'

rithishapadmanabh commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,6 +4,28 @@ include::_attributes.adoc[]
+
+To use the API mocking service for an asset:
+
+. Click *All Types* > *REST APIs* and choose any API.
```
This is not fully accurate. "All types" assumes the user is using the old search. With the new Search it is 'Any type'.

JennyHajee replied:
fixed

rithishapadmanabh commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,6 +4,28 @@ include::_attributes.adoc[]
+. Click *All Types* > *REST APIs* and choose any API.
+. Click the asset and view the API functions in the left navigation area.
+. Click a function to show more information about that function.
+. At the right side of the screen, test the API by entering sample values to send to the API and get responses from the API.
```
Instead of the right side of the screen would 'the right navigation area' or the 'right panel on the screen' work?

kbostrom replied:
Do we have to call this "the right side" at all: is this just the space to the right of the nav menu? Or is there some content between the left nav and this testing area? I guess I'd need to see it to know what to call it if it hasn't been provided a label....
Maybe something like "In the testing area, enter sample values to test the API response."?

rithishapadmanabh commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,6 +4,28 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
+== View Assets
```
How is this section intended to be different from Explore Asset Information section?

kbostrom replied:
Why is there no text between = Asset Details and == View Assets?
The container topic should provide background (what we mean by asset details and why the audience cares) and refer to the content included within: in this case, viewing (and testing? why is testing here?)

rithishapadmanabh commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,6 +4,28 @@ include::_attributes.adoc[]
+
+== Test APIs by Using the Mocking Service
+
+To use the API mocking service for an asset:
```
There is a mention of using the Mocking Service under the next section i.e. the Explore Asset Information section as well. Both of these seem redundant..

rithishapadmanabh commented on modules/ROOT/pages/index.adoc:
```diff
@@ -5,71 +5,19 @@ endif::[]
-
-Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API Groups, and policies), and Runtime Manager (APIs), and you can publish APIs from Design Center to Exchange.
-
-MuleSoft certifies its public content and reviews and approves partner connectors. All private content is internal to an organization and not visible outside that organization. Customers are responsible for any private content they publish within their Exchange instance.
```
I would want to retain some version of this. something on the lines of:

Assets under the 'Provided by Mulesoft' organization are created by Mulesoft and its partners, and are certified by Mulesoft. These assets are publicly available to all customers as well. However, all assets that are created by an organization and published into the organization's instance of Exchange, are private and not visible outside that organization.... Customers are responsible for any private content they publish within their organization's Exchange instance.

JennyHajee replied:
Done. added this to line 18 in the index.adoc.

rithishapadmanabh commented on modules/ROOT/pages/index.adoc:
```diff
@@ -5,71 +5,19 @@ endif::[]
 :page-aliases: general:getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc
 
-Anypoint Exchange provides the benefit of being able to discover, share, and incorporate assets and resources into your applications. Anypoint Exchange helps you create API developer portals, view and test APIs, simulate data to APIs (Mocking Service), create assets, and use API Notebooks to describe and test API functions.
+Anypoint Exchange is your curated catalog of APIs and other assets, including API groups, API specification fragments, policies, and more.
```
Can we elaborate a bit more on the value and specify assets beyond just APIs as well. Something on the lines of "Anypoint Exchange is a catalog of reusable assets such as APIs, Policies, API spec fragments, API groups as well as other integration assets such as connectors, templates, examples and custom assets. Anypoint Exchange can be used to catalog (publish), share, discover, learn about, and reuse assets within your organization, to drive collaboration, boost productivity and promote standards."

JennyHajee replied:
done

JennyHajee committed:
Update asset-details.adoc

removed "private" when referring to Exchange. Changed, "Click helpful links in the navigation area for links to documentation" to "View the documentation and metadata to learn more about the asset".

JennyHajee committed:
Update asset-details.adoc

JennyHajee committed:
Update index.adoc

Updated intro paragraph per Rithisha.

JennyHajee committed:
index.adoc

changes per Rithisha

JennyHajee committed:
Merge branch 'latest' into docs-10084-landing-page

kbostrom commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
 
+== View Assets
+
+To view assets:
```
Without my having any context/knowledge of preceding narrative, I wonder if we can make this introduction to the steps more robust than just essentially repeating the title, something like "Viewing <which?> assets enables you to <work more productively in this/these particular ways>."
Just an idea; it is OK as-is, but could possibly be better.

kbostrom commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
+
+To view assets:
+
+. From the public https://www.anypoint.mulesoft.com/exchange/[Exchange], click an asset and view its description. See the Explore Asset Information section for more information about viewing an asset's detail.
```
Can we not say "From Exchange, click..." rather than specifying that it is public?
If we keep "public," I suggest "From a public instance of Exchange,..." because we do not want to use the article "the" before the proper noun Exchange (without a common noun following: the <Exchange> <common noun>).

kbostrom replied:
Do we really need to refer the audience to some other location in this step? 
Maybe it should be a prerequisite to the task that you already are familiar with asset details?

kbostrom commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
+To view assets:
+
+. From the public https://www.anypoint.mulesoft.com/exchange/[Exchange], click an asset and view its description. See the Explore Asset Information section for more information about viewing an asset's detail.
+. View the documentation and metadata to learn more about the asset.
```
How is this different from "view its description" in Step 1?
I think this is unnecessary repetition of what Step 1 implies, and "the documentation" actually confused me: are you referring to the UI or to the text you just referred to in this (unnecessary) second step?

kbostrom commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
+. Click *Login* to view assets published into Exchange by your organization.
+. After you log in, click *Public Portal* in your organization to view public APIs available to anyone.
+
+After you log in to Exchange, you can create assets.
```
So...is this topic really about logging in and viewing, rather than just viewing? How do the previous steps relate to the ability to create assets?

kbostrom replied:
...and should creating assets directly follow viewing assets?

kbostrom commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
+
+. From the public https://www.anypoint.mulesoft.com/exchange/[Exchange], click an asset and view its description. See the Explore Asset Information section for more information about viewing an asset's detail.
+. View the documentation and metadata to learn more about the asset.
+. Click *Login* to view assets published into Exchange by your organization.
```
So does this task include viewing both public and private assets? If so, we might want to specify that in the task intro.
Does this step actually FOLLOW the previous or is it a separate action (viewing private vs. viewing public)?

kbostrom commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
+. From the public https://www.anypoint.mulesoft.com/exchange/[Exchange], click an asset and view its description. See the Explore Asset Information section for more information about viewing an asset's detail.
+. View the documentation and metadata to learn more about the asset.
+. Click *Login* to view assets published into Exchange by your organization.
+. After you log in, click *Public Portal* in your organization to view public APIs available to anyone.
```
Wait, I'm confused (and please bear with me if this is just not my familiarity with the product):
What is the actual task here? 
Is it viewing public or private assets?
Is the flow really this:
1. Navigate to Exchange to select and view the details of public assets.
2. Log in to Exchange and click *Public Portal* to view the details of assets that belong to your organization.
Why are assets that belong to your org. "available to anyone" in your org's "public portal"?

kbostrom commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
+
+== Test APIs by Using the Mocking Service
+
+To use the API mocking service for an asset:
```
Is this more precise:
```suggestion
To use the API mocking service to test an asset:
```

kbostrom commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
+
+To use the API mocking service for an asset:
+
+. Click *Any Types* > *REST APIs* and choose any API.
```
From what starting point?
```suggestion
. From <screen on which these choices are displayed>, click *Any Types* > *REST APIs* and choose any API.
```

kbostrom commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
+To use the API mocking service for an asset:
+
+. Click *Any Types* > *REST APIs* and choose any API.
+. Click the asset and view the API functions in the left navigation area.
```
```suggestion
. Click the asset and view the list of API functions in the left navigation area.
```

kbostrom commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
+
+After you log in to Exchange, you can create assets.
+
+== Test APIs by Using the Mocking Service
```
```suggestion
== Test Assets by Using the API Mocking Service
```

JennyHajee replied:
can only test APIs

kbostrom commented on modules/ROOT/pages/asset-details.adoc:
```diff
@@ -4,9 +4,31 @@ include::_attributes.adoc[]
+. Click a function to show more information about that function.
+. At the right side of the screen, test the API by entering sample values to send to the API and get responses from the API.
+
+== Explore Asset Information
```
I have the same question as your SME reviewer: how is this different from the goal of View Assets?
And why is testing assets included in the container "Asset Details"?

Could it be that you really want an org like this:

 = Exploring Assets (all level-1 titles should use gerunds: see our templates)
 text about why audience wants/needs to do this: advantages
  == View Asset Information (all topics with steps need an imperative verb; see templates)
  intro to task, if any: maybe about info you see before and after logging in?
  steps, ending with clicking an asset
  what you see displayed after clicking an asset
 == View Assets by Business Group....
 == View REST API....
 
 Followed by a separate container...

  = Testing Assets (gerund)
  blah blah blah as above
   == Test Assets by Using the API Mocking System (imperative)
 blah blah blah steps
  == Test [Assets or API?] by Using API Notebook (imperative)
 blah blah blah steps

kbostrom reviewed (dismissed):
CU/CE asset-details.adoc

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
+
+Exchange supports these asset types:
+
+* Connectors - Packaged connectivity to an endpoint developed and deployed on Anypoint Platform with third-party APIs and standard integration protocols
```
GLOBALLY replace space-hyphen-space with colon-space, per style ref.

kbostrom replied:
```suggestion
* Connectors: Packaged connectivity to an endpoint, developed and deployed on Anypoint Platform by using third-party APIs and standard integration protocols
```

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
+Use connectors within your application's flows to send and receive data using a protocol or specific API. Anypoint Studio comes with many bundled connectors, and Exchange has many more.
+* Templates - Packaged integration patterns built on best practices to address common use cases
++
+You can add your information such as user credentials to complete the template's use case or solution, and you can customize or extend templates as needed.
```
```suggestion
You can add your information, such as user credentials, to complete the template's use case or solution, and you can customize or extend templates as needed.
```

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
+* Templates - Packaged integration patterns built on best practices to address common use cases
++
+You can add your information such as user credentials to complete the template's use case or solution, and you can customize or extend templates as needed.
+* Examples - Applications that are ready to run in Anypoint Studio and demonstrate a use case or solution
```
```suggestion
* Examples - Applications that are ready to run in Anypoint Studio and that demonstrate a use case or solution
```
better:
```suggestion
* Examples - Applications that demonstrate a use case or solution and are ready to run in Anypoint Studio.
```

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
++
+You can add your information such as user credentials to complete the template's use case or solution, and you can customize or extend templates as needed.
+* Examples - Applications that are ready to run in Anypoint Studio and demonstrate a use case or solution
+* Policies - Configuration modules to extend the functionality of an API and enforce capabilities such as security
```
```suggestion
* Policies - Configuration modules that extend the functionality of an API and enforce capabilities such as security
```

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
+* Policies - Configuration modules to extend the functionality of an API and enforce capabilities such as security
+* REST APIs - A RAML file or OAS file that specifies an API
++
+These APIs can be referenced by an HTTP Request connector to expose metadata to Anypoint Studio.
```
```suggestion
REST APIs can be referenced by an HTTP Request connector to expose metadata to Anypoint Studio.
```

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
+* REST APIs - A RAML file or OAS file that specifies an API
++
+These APIs can be referenced by an HTTP Request connector to expose metadata to Anypoint Studio.
+* AsyncAPI specifications - An AsyncAPI specification file that specifies an event-driven API
```
plural "specifications" but singular "specification file"?
How about
```suggestion
* AsyncAPI specifications: AsyncAPI files that specify event-driven APIs
```

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
+* HTTP APIs - A placeholder for an endpoint for use by private Exchange users who want to manage the endpoint with API Manager
+* API Groups - A set of APIs bundled into a single asset
++
+Instead of requesting access to multiple APIs to satisfy a use case, a developer can access the group in one step.
```
```suggestion
Instead of making multiple requests for access to multiple APIs to satisfy a use case, a developer makes one request for one bundle of APIs.
```

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
+You can add your information such as user credentials to complete the template's use case or solution, and you can customize or extend templates as needed.
+* Examples - Applications that are ready to run in Anypoint Studio and demonstrate a use case or solution
+* Policies - Configuration modules to extend the functionality of an API and enforce capabilities such as security
+* REST APIs - A RAML file or OAS file that specifies an API
```
```suggestion
* REST APIs: RAML or OAS files that specify an API
```

kbostrom replied:
GLOBALLY ensure that the type and the description match in terms of number (singular or plural).
You can modify either element when they are not, as long as you modify all list items in a consistent way.

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
++
+These APIs can be referenced by an HTTP Request connector to expose metadata to Anypoint Studio.
+* AsyncAPI specifications - An AsyncAPI specification file that specifies an event-driven API
+* SOAP APIs - A WSDL file that specifies an API
+* HTTP APIs - A placeholder for an endpoint for use by private Exchange users who want to manage the endpoint with API Manager
+* API Groups - A set of APIs bundled into a single asset
```
ditto numerical agreement

kbostrom replied:
Also capitalization: API Groups > API groups

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
+* API Groups - A set of APIs bundled into a single asset
++
+Instead of requesting access to multiple APIs to satisfy a use case, a developer can access the group in one step.
+* API Specification Fragments - A RAML document that has a version and an identifier, but is not in itself a complete RAML specification
```
numerical agreement

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
+Instead of requesting access to multiple APIs to satisfy a use case, a developer can access the group in one step.
+* API Specification Fragments - A RAML document that has a version and an identifier, but is not in itself a complete RAML specification
++
+API specification fragments are also known as RAML fragments.
```
"called," "known as", and similar introductions to names or terms are formatted using italic:
```suggestion
API specification fragments are also known as _RAML fragments_.
```

kbostrom commented on modules/ROOT/pages/asset-types.adoc:
```diff
@@ -0,0 +1,29 @@
+* API Specification Fragments - A RAML document that has a version and an identifier, but is not in itself a complete RAML specification
++
+API specification fragments are also known as RAML fragments.
+* Custom - A description and an optional file to explain aspects of your system, to provide instructional videos, or to describe product or organizational documentation
```
```suggestion
* Custom: A description and, optionally, a file that explains aspects of your system, provides instructional videos, or describes product or organizational documentation
```

kbostrom commented on modules/ROOT/pages/index.adoc:
```diff
@@ -5,71 +5,19 @@ endif::[]
 :page-aliases: general:getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc
 
-Anypoint Exchange provides the benefit of being able to discover, share, and incorporate assets and resources into your applications. Anypoint Exchange helps you create API developer portals, view and test APIs, simulate data to APIs (Mocking Service), create assets, and use API Notebooks to describe and test API functions.
+Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API spec fragments, API groups, templates, examples, custom assets, and integration assets such as connectors. You can catalog (publish), share, discover, learn about, and reuse assets within your organization that drive collaboration, boost productivity, and promote standards.
```
```suggestion
Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API specification fragments, API groups, templates, examples, custom assets, and integration assets such as connectors. You can catalog (or _publish_), share, discover, learn about, and reuse assets within your organization to facilitate collaboration, boost productivity, and promote standards.
```

JennyHajee committed:
Update modules/ROOT/pages/asset-details.adoc

Co-authored-by: Kim Bostrom <18537445+kbostrom@users.noreply.github.com>

kbostrom commented on modules/ROOT/pages/index.adoc:
```diff
@@ -5,71 +5,19 @@ endif::[]
-. From the public https://www.anypoint.mulesoft.com/exchange/[Exchange], click an asset and view its description.
-. Click Login to view a private Exchange for your organization.
-. After you log in, click Public Portal in your organization to view public APIs available to anyone.
+Use Exchange to discover, share, and incorporate assets and resources into your applications. Exchange helps you create API developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.
```
```suggestion
Use Exchange to discover, share, and incorporate assets into your applications. Exchange helps you create API developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.
```

kbostrom commented on modules/ROOT/pages/index.adoc:
```diff
@@ -5,71 +5,19 @@ endif::[]
-This page applies to the current version, Exchange 2.
-
-Documentation on Exchange 1 is at xref:about-exchange1.adoc[About Exchange 1 (Old Version)]. We encourage Exchange 1 users to xref:migrate.adoc[migrate] to Exchange 2.
+Assets under the *Provided by MuleSoft* organization are created by MuleSoft and its partners, and are certified by Mulesoft. These assets are publicly available to all customers as well. However, all assets that are created by an organization and published into the organization's instance of Exchange, are private and not visible outside that organizationCustomers are responsible for any private  ...
```
```suggestion
Assets displayed as *Provided by MuleSoft* are created by MuleSoft and its partners and certified by MuleSoft. These assets are publicly available to all customers. However, any assets created by an organization and published into that organization's instance of Exchange are private: not visible to customers outside of that organization. Organization customers are responsible for any private assets they publish within their Exchange instance.
```

kbostrom commented on modules/ROOT/pages/index.adoc:
```diff
@@ -5,71 +5,19 @@ endif::[]
+Assets under the *Provided by MuleSoft* organization are created by MuleSoft and its partners, and are certified by Mulesoft. These assets are publicly available to all customers as well. However, all assets that are created by an organization and published into the organization's instance of Exchange, are private and not visible outside that organizationCustomers are responsible for any private  ...
 
-The full version history is available in the xref:release-notes::exchange/anypoint-exchange-release-notes.adoc[Anypoint Exchange Release Notes].
+This page applies to the current version, Exchange 2. Documentation on Exchange 1 is at xref:about-exchange1.adoc[About Exchange 1 (Old Version)]. We encourage Exchange 1 users to xref:migrate.adoc[migrate] to Exchange 2. The full version history is available in the xref:release-notes::exchange/anypoint-exchange-release-notes.adoc[Anypoint Exchange Release Notes].
```
We never overtly refer to units of information as pages, topics, subtopics, or any similar terms.
I suggest incorporating this information at the very beginning of the text introducing the page, something like "Before <blah blah blah>, you must be running Exchange <v.2.0?> or later.
OR
 Incorporate it as a Note at the end of that introduction:
```suggestion
NOTE: This information applies to the Exchange [v2.0 and later]current version. To get more information about Anypoint Exchange versioning, see xref:release-notes::exchange/anypoint-exchange-release-notes.adoc[Anypoint Exchange Release Notes].
```

kbostrom reviewed (approved):
RA
